### PR TITLE
DOCS-706: Update transformation order

### PR DESCRIPTION
**Pull request by npentrel**

### CODE REVIEW

File: docs/components/camera/transform.md [hunk 1]
```diff
@@ -74,59 +74,66 @@ The following attributes are available for `transform` views:
...
+        "low_threshold_pct": float,
+        "blur_radius_px": float
+    }
 }
 ```
 
 **Attributes:**
 
-- None
+- `high_threshold_pct`: The high threshold of ??? : between 0.0 - 1.0.
```
Comment (sguequierre) [inline docs/components/camera/transform.md:115]: ??? I think needs to be clarified here and below! Sorry as I don't know what it is either

**Review by sguequierre — CHANGES_REQUESTED** [review]
Would LGTM when ??? is addressed!

File: docs/components/camera/transform.md (hunk 1 above)
Comment (npentrel) [reply docs/components/camera/transform.md:115]: Oh amazing 😂

Comment (npentrel) [inline docs/components/camera/transform.md:115]:
```suggestion
- `high_threshold_pct`: The high threshold value: between 0.0 - 1.0.
```

File: docs/components/camera/transform.md [hunk 2]
```diff
@@ -74,59 +74,66 @@ The following attributes are available for `transform` views:
...
+- `low_threshold_pct`: The low threshold of ??? : between 0.0 - 1.0.
```
Comment (npentrel) [inline docs/components/camera/transform.md:116]:
```suggestion
- `low_threshold_pct`: The low threshold value: between 0.0 - 1.0.
```

File: docs/components/camera/transform.md (hunk 1 above)
Comment (npentrel) [reply docs/components/camera/transform.md:115]: basing on this https://en.wikipedia.org/wiki/Canny_edge_detector

**Review by sguequierre — APPROVED** [review]

**Pull request merged**